README.md

```diff
@@ -0,0 +1,21 @@
+# foo-config-loader
+
+A config loader for the Foo project.
+
+
+# Project tasks
+
+## In order to setup a local env run...:
+
+    virtualenv create -p python3.6 venv/
+    ./venv/bin/pip install pip-tools
+    ./venv/bin/pip install -r requirements.txt -r requirements-test.txt
+
+## In order to update the requirements run...:
+
+    ./venv/bin/pip-compile requirements.in
+    ./venv/bin/pip-compile requirements-test.in
+
+## In order to execute tests run...:
+
+    env PYTHONPATH=. ./venv/bin/pytest test
```

Comment:
similar to the idea of preparing `setup_dev_env.sh`. How about creating a single file that could be used to running tests?
It could set `PYTHONPATH` automatically and could save some typing.
WDYT?

Reply:
Also how about defining a way of running only specified tests, and not all of them?